cmd/thanos/query.go

```diff
@@ -470,6 +484,24 @@ func runQuery(
 	return nil
 }
 
+func runtimeInfo() v1.RuntimeInfo {
+	cwd, err := os.Getwd()
+	if err != nil {
+		cwd = "<error retrieving current working directory>"
```

Comment:
should we log the error? Or maybe put the error in the returned string?

Reply:
I think logging it would make more sense. I got this from https://github.com/thanos-io/thanos/blob/master/pkg/ui/query.go#L47-L50

Should we log the error there too? And I guess the log-level should be `warn`, or do we need to use log-level error for this?

Reply:
yes warn sounds good IMO, it as it doesn't cause the entire request to fail

Reply:
Done :+1: